frontend/src/data/constants.ts

```diff
@@ -60,6 +61,13 @@ export const PALLET_PAYMENT_STATUS_OPTIONS = [
   },
 ]
 
+export const LINE_ITEM_CONTAINER_OPTIONS = [
+  { label: 'Not set', value: LineItemContainerType.Unset },
+  { label: 'Box', value: LineItemContainerType.Box },
+  { label: 'Bulk bag', value: LineItemContainerType.BulkBag },
+  { label: 'Full pallet', value: LineItemContainerType.FullPallet },
+] as const
```

Comment:
I recently learned that `as const` forces a variable to be read-only! This means that the array can't be modified.

Reply:
🤯